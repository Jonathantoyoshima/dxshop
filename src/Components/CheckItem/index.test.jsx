import React from 'react';
import renderer from 'react-test-renderer';
import CheckItems from './index';

test('Render Build Check Items Snapshot', () => {
	const component = renderer.create(<CheckItems />);
	let tree = component.toJSON();
	expect(tree).toMatchSnapshot();
});
