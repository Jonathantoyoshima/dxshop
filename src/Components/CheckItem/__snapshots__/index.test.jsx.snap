// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Build CheckItem Snapshot 1`] = `ShallowWrapper {}`;
